smokes/e2e/pages/about.coffee

```diff
@@ -0,0 +1,29 @@
+# this file will contains the different generic functions which
+# will be called by the different tests
+# inspired by this methodology
+# http://www.lindstromhenrik.com/using-protractor-with-coffeescript/
+
+class aboutPage
+    constructor: (builder) ->
+        @builder = builder
+
+    goAbout: () ->
+        browser.get('#/about')
+
+    checkAboutPage: () ->
+        expect(browser.getCurrentUrl()).toContain('#/about')
+
+    checkBuildbotTitle: () ->
+        aboutTitle = element.all(By.css('h2')).first()
+        expect(aboutTitle.getText()).toContain('About this')
+        expect(aboutTitle.getText()).toContain('buildbot')
+
+    checkConfigTitle: () ->
+        configurationTitle = element.all(By.css('h2')).get(1)
+        expect(configurationTitle.getText()).toContain('Configuration')
+
+    checkDependenciesTitle: () ->
+        dependenciesTitle = element.all(By.css('h2')).get(2)
+        expect(configurationTitle.getText()).toContain('Javascript Dependencies')
```

Comment:
please always make sure you run your tests:
 
      Failed: configurationTitle is not defined